Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -851,6 +851,20 @@ class PaymentSheetStandardLPMUITests: PaymentSheetUITestCase {
         _testUSBankAccount(mode: .setup, integrationType: .normal)
     }
 
+    func testPaymentIntent_instantDebits() {
+        // UI tests need more fixes to work for iOS 17
+        if #unavailable(iOS 17.0) {
```

Comment:
I'm trying to merge this PR ASAP as it also introduces the new "supported payment method" textfield (which would be important for bug bashes)

right now Bitrise runs iOS 16 anyway: `destination: platform=iOS Simulator,name=iPhone 12 mini,OS=16.4`

I am looking into fixing iOS 17 UI tests, and spent a long time trying to fix it already. The UI tests are struggling with selecting certain UI elements in the Stripe.js WebView (may or may not need Stripe.js changes)